#! /bin/sh /usr/share/dpatch/dpatch-run
## link_list_error_fix.dpatch by  <jame@rocasa.us>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Line 133 in src/xpdev/link_list.c was causing the following error during a compile:
## DP: link_list.c:133: error: incompatible types in assignment
## DP: Commented that line out, as did upstream (reference revsion 1.37 of that file in
## DP: cvs.synchro.net).
@DPATCH@
diff -urNad synchronet_3.14a~/src/xpdev/link_list.c synchronet_3.14a/src/xpdev/link_list.c
--- synchronet_3.14a~/src/xpdev/link_list.c	2006-05-29 03:32:43.000000000 -0400
+++ synchronet_3.14a/src/xpdev/link_list.c	2009-10-18 13:15:23.734574160 -0400
@@ -130,7 +130,7 @@
 	if(list->flags&LINK_LIST_SEMAPHORE) {
 		while(sem_destroy(&list->sem)==-1 && errno==EBUSY)
 			SLEEP(1);
-		list->sem=NULL;
+		// list->sem=NULL; 
 	}
 #endif
 
